src/pages/ecdn-api-getting-started copy.mdx

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
We already have a Getting Started page. Please check why this `...copy.mdx` created!

Reply:
I removed it. That page haven't changed I wasn't aware of that copy.